cert-data.yml

```diff
@@ -1,4 +1,179 @@
 ---
+### 4_programmatic Default course
+Default/OpenEdX/Certificate: &cert_defaults
+  VERSION: 4_programmatic
+  TEMPLATEFILE: False
+  instructors:
+    - instructor: &instructor_1
+        x_position: 475
+        y_position: 140
+    - instructor: &instructor_2
+        x_position: 475
+        y_position: 220
+    - instructor: &instructor_3
+        x_position: 475
+        y_position: 300
+    - instructor: &instructor_4
+        x_position: 475
+        y_position: 380
+    - instructor: &instructor_5
+        x_position: 275
+        y_position: 140
+    - instructor: &instructor_6
+        x_position: 75
+        y_position: 140
+  instructor_template:
+    # ordered list of steps to render the instructor signature block, bottom to top
+    - text:
+        string: "school"
+        font_size: 10
+        leading: 10
+        text_color: [.18,.18,.18]
+        alignment: TA_RIGHT
+        width: 250
+        height: 12
+    - text:
+        string: "title_2"
+        font_size: 10
+        leading: 10
+        text_color: [.18,.18,.18]
+        alignment: TA_RIGHT
+        width: 250
+        height: 11
+    - text:
+        string: "title"
+        font_size: 10
+        leading: 10
+        text_color: [.18,.18,.18]
+        alignment: TA_RIGHT
+        width: 250
+        height: 11
+    - text:
+        string: "name"
+        font_size: 12
+        leading: 12
+        text_color: [.18,.18,.18]
+        alignment: TA_RIGHT
+        width: 250
+        height: 12
+    - image:
+        file: "signature_file"
```

Comment:
no this is the template, so it takes the file path provided in the course description further down in cert-data